.github/workflows/operator-integration-test.yml

```diff
@@ -0,0 +1,91 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: operator Integration Test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [ opened, reopened ]
```

Comment:
can we include `synchronize, ready_for_review` ? 

This should trigger the build in this PR